server/app.js

```diff
@@ -13,19 +13,19 @@ const clientSecret = '<client_secret>'
 
 const columnUrl = "https://sandbox.columnapi.com/v1/exp/initialize_tax_filing"
 
-// An endpoint to retrieve a user token from column tax. This token will be used in the
+// An endpoint to retrieve a user url from column tax. This url will be used in the
 // frontend application to open to column tax SDK.
```

Comment:
nit: "to open the Column Tax SDK", and capitalizing both references to CT?

Reply:
Done!